### PR TITLE
responsible data addition

### PR DESCRIPTION
**Pull request by n-orlowski**
Content is in - need to figure out how you want to add footnotes. I also changed the title case to sentence case as per our style guide

### CODE REVIEW

**Review by helizabetholsen — APPROVED** [review]
I did a quick update to the "Responsible Data" description but, other than that, looks awesome. Thanks, Nicole. I think we can leave the footnotes out for now and revisit later?

**Comment by n-orlowski** [comment]
Thanks @helizabetholsen ! Will wait on @michaelkohn to review before merging - we may want to rethink the "Responsible Data" section title as we have one for "[Data](https://docs.communityhealthtoolkit.org/apps/guides/data/)" for data management

**Comment by michaelkohn** [comment]
Is it me, or does it seem weird that Medic specific policies are on a documentation site for the CHT?

**Comment by helizabetholsen** [comment]
@michaelkohn thanks for this comment. @isaacholeman suggested that we have our Responsible Data policy on the CHT docs site as an example of the CHT's commitment to responsible data - and as an example to other organizations. 

Do you think we could revise the header on this section to add additional context to make it less weird?

**Comment by michaelkohn** [comment]
I would have thought the CHT doc site page would have looked something like:

- as CHT implementer, you should have a responsible data policy
- your responsible data policy should include x,y,z
- here's an example (link to Medic's policy on mm.org)

**Comment by garethbowen** [comment]
@helizabetholsen @michaelkohn @isaacholeman Just circling back here - what content do we want to include?

**Comment by michaelkohn** [comment]
My opinion hasn't changed, but I haven't heard any other updates. I'll reach out on Slack to see if there is a plan to include it on our website.

**Comment by helizabetholsen** [comment]
@michaelkohn I can work on this today but I think we can reframe slightly to be something of a middle ground. I don't think Medic is in the position to tell CHT implementers what to include in their responsible data policies but we can highlight our commitment to responsible data practices and include examples of our policies, if we all agree.

This would look something like:
- A commitment to responsible data practices is an important component of the CHT and CHT community
- At Medic Mobile, we have a responsible data policy that we'd like to share with the broader community
- If you'd like to learn more about the different SOPs that help us operationalize this policy, please reach out. 

What do you think? cc @isaacholeman

**Comment by isaacholeman** [comment]
Looks like we all agree that there should first be something like a 'Responsible Data Practices' page, and that Medic's privacy and data protection policy would be linked from that page and described as an illustrative example. @michaelkohn and @helizabetholsen's outlines are a little bit different but seem similar enough to integrate. I'd also add that let's spend a few sentences describing what we mean by a responsible data practice and why it matters. This can borrow text from the paper we just wrote with Beth and Beth from human-centered design at UW.

**Comment by garethbowen** [comment]
@n-orlowski Can you follow up to get this merged?

**Comment by n-orlowski** [comment]
@garethbowen, confirmed that @helizabetholsen is still workshopping this and content should be ready next week

**Comment by helizabetholsen** [comment]
@garethbowen I'm the one holding this up. I need to do a final review and edit of the content and I just haven't had the time. I will prioritize for before the end of the week!

File: content/en/apps/guides/privacy/privacy.md
Comment (andy5995) [inline content/en/apps/guides/privacy/privacy.md:37]: This bullet point ends in a comma, and the second with a period. Is that what's intended?

I'm actually not sure that either of these two points require punctuation at the end because they're not complete sentences.

File: content/en/apps/guides/privacy/privacy.md
Comment (andy5995) [inline content/en/apps/guides/privacy/privacy.md:33]:
```suggestion
One of Medic Mobile’s mission goals is to advance good health and human flourishing by collaborating with hard-to-reach communities when building open source technology. We value humanity, creativity, initiative, solidarity, and openness. Being responsible stewards of people’s data is critically important to our mission. 
```

The "with and for" sounded awkward to me so I'm offering this an alternative.

File: content/en/apps/guides/privacy/privacy.md
Comment (abbyad) [inline content/en/apps/guides/privacy/privacy.md:38]:
```suggestion
1. Prioritising people’s rights to consent, privacy, security and ownership when using data in social change and advocacy efforts
2. Implementing values and practices of transparency and openness
```

File: content/en/apps/guides/privacy/_index.md
Comment (abbyad) [inline content/en/apps/guides/privacy/_index.md:6]:
```suggestion
  This series of guides is related to the Community Health Toolkit's commitment to responsible data use, as well as a space for policies and templates to be shared with the broader community. 
```

File: content/en/apps/guides/privacy/_index.md
Comment (abbyad) [inline content/en/apps/guides/privacy/_index.md:3]:
```suggestion
title: "Privacy"
linkTitle: "Privacy"
```

File: content/en/apps/guides/privacy/_index.md
Comment (abbyad) [reply content/en/apps/guides/privacy/_index.md:6]: Reworded to match style of other descriptions
```suggestion
  Policies and templates for privacy and responsible data use 
```

File: content/en/apps/guides/privacy/privacy.md
Comment (abbyad) [inline content/en/apps/guides/privacy/privacy.md:9]:
```suggestion
title: "Privacy & Data Protection Policy"
linkTitle: "Policy"
weight: 
description: >
  Medic’s Privacy and Data Protection Policy, which applies to the Community Health Toolkit
keywords: 
relatedContent: >
  apps/reference/app-settings/patient_reports
```

File: content/en/apps/guides/privacy/privacy.md
Comment (abbyad) [inline content/en/apps/guides/privacy/privacy.md:16]:
```suggestion
If you have any questions, please reach out to our Data Protection Officer at support@medic.org. 
```

File: content/en/apps/guides/privacy/privacy.md
Comment (abbyad) [inline content/en/apps/guides/privacy/privacy.md:6]:
```suggestion
  This is the current version of Medic’s Privacy and Data Protection Policy, which applies to the Community Health Toolkit.
```

**Comment by abbyad** [comment]
Seems as though the [guide on setting up privacy policies in the app](https://github.com/medic/cht-docs/blob/54f4b6d86c7b10f81b071652a5a9244281d28cfd/content/en/apps/guides/security/privacy-policy.md) should be moved to this section.

File: content/en/apps/guides/privacy/privacy-policy.md
Comment (abbyad) [inline content/en/apps/guides/privacy/privacy-policy.md:2]:
```suggestion
title: "Adding Privacy Policies to Apps"
```

**Comment by abbyad** [comment]
@helizabetholsen I'm rerequesting your review given the structural/heavy changes.

**Review by helizabetholsen — APPROVED** [review]
This looks great @abbyad! Thanks for checking in as you revised 😊✨

**Comment by abbyad** [comment]
Link checking is no longer done in master so should pass when merged. Links are being fixed in #476.

**Pull request merged**
